src/jsonapi-datastore.js

```diff
@@ -11,6 +11,7 @@
     this.id = id;
     this._type = type;
     this._attributes = [];
+    this._associations = [];
```

Comment:
That is correct. In our application we called it _references, but for the PR I changed it to associations. I'm not sure the best name for that property. Maybe _inverse_associations?